commands/mr/close/mr_close.go

```diff
@@ -18,6 +19,12 @@ func NewCmdClose(f *cmdutils.Factory) *cobra.Command {
 		Short: `Close merge requests`,
 		Long:  ``,
 		Args:  cobra.MaximumNArgs(1),
+		Example: heredoc.Doc(`
+			$ glab mr close 1
+			$ glab mr close branch
+			$ glab mr close username:branch
+			$ glab mr close branch -R another/repo
```

Comment:
Also without supplying any argument, it finds the mr for the current branch
```suggestion
		Example: heredoc.Doc(`
			$ glab mr close   # Closes mr for current branch
			$ glab mr close 1
			$ glab mr close branch
			$ glab mr close username:branch
			$ glab mr close branch -R another/repo
```